### ISSUE TITLE
Unexpected constant violation (VariableDeclarator)

### ISSUE THREAD

**Issue by youdie323323** [j4k0xb/webcrack#80]
### Describe the bug

The error appear when trying to deobfuscate code

### Expected Behaviour

Should deobfuscate correctly

### Code

```JavaScript
https://qu.ax/zEJG.txt
```

### Logs

```Text
Error: Unexpected constant violation (VariableDeclarator): {currentPoint:a,rX:o,rY:i,sweepFlag:c,xAxisRotation:l,centp:v,a1:y,ad:d}=Z[s(1830)](f)
    at webcrack.worker-DAM0XNvR.js:71:204515
    at Array.forEach (<anonymous>)
    at En (webcrack.worker-DAM0XNvR.js:71:203971)
    at BJ (webcrack.worker-DAM0XNvR.js:71:212326)
    at Object.run (webcrack.worker-DAM0XNvR.js:76:599)
    at JT (webcrack.worker-DAM0XNvR.js:71:204958)
    at Array.<anonymous> (webcrack.worker-DAM0XNvR.js:78:38083)
    at IY (webcrack.worker-DAM0XNvR.js:78:38397)
    at async self.onmessage (webcrack.worker-DAM0XNvR.js:78:39020)
```

**Comment by j4k0xb** [comment]
It's a babel bug, have to wait until it's fixed: https://github.com/babel/babel/issues/16512

**Comment by TheVisual** [comment]
> It's a babel bug, have to wait until it's fixed: [babel/babel#16512](https://github.com/babel/babel/issues/16512)

its fixed they closed it 8 hours ago

**Comment by j4k0xb** [comment]
not released yet though
